crates/cli/src/hooks/go.rs

```diff
@@ -0,0 +1,32 @@
+use crate::shell;
+use clap_complete::Shell;
+use log::info;
+use proto_core::{color, ProtoError};
+use rustc_hash::FxHashMap;
+
+pub fn post_install(passthrough: &[String]) -> Result<(), ProtoError> {
+    if passthrough.contains(&"--no-gobin".to_string()) {
+        return Ok(());
+    }
+
+    let Some(shell) = Shell::from_env() else {
+        return Ok(());
+    };
+
+    let env_vars = FxHashMap::from_iter([
+        ("GOBIN".to_string(), "$HOME/go/bin".to_string()),
+        ("PATH".to_string(), "$GOBIN".to_string()),
```

Comment:
Thought his was setting the PATH to GOBIN. A little confusing that's a _special_ key, but I get it

Reply:
I feel that. Felt the same while writing it but this was the simplest approach.